machineconfiguration/v1/containerruntimeconfig.crd.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: containerruntimeconfigs.machineconfiguration.openshift.io
+  labels:
+    "openshift.io/operator-managed": ""
+spec:
+  group: machineconfiguration.openshift.io
+  names:
+    kind: ContainerRuntimeConfig
+    listKind: ContainerRuntimeConfigList
+    plural: containerruntimeconfigs
+    singular: containerruntimeconfig
+    shortNames:
+    - ctrcfg
```

Comment:
is this already present in your CRD manifest today?  We discourage shortnames on resources that are infrequently used by cluster-admins.

Reply:
yes, already present